.github/workflows/build-onvif-video-broker-container.yml

```
@@ -49,13 +49,14 @@ jobs:
       uses: docker/setup-buildx-action@v3
 
     - name: Login to GitHub Container Registry
+      if: github.event_name == 'push' || github.event_name == 'release'
       uses: docker/login-action@v3
       with:
         registry: ghcr.io

```

I wonder if we need to specify the fully qualified registry if we are not specifying the GITHUB token as the [docs recommend](https://github.com/docker/login-action#github-container-registry).
```suggestion
        registry: ghcr.io/project-akri
```